src/azure-cli/azure/cli/command_modules/storage/__init__.py

```diff
@@ -111,7 +111,15 @@ def register_path_argument(self, default_file_param=None, options_list=None):
 
     def register_source_uri_arguments(self, validator, blob_only=False):
         self.argument('copy_source', options_list=('--source-uri', '-u'), validator=validator, required=False,
-                      arg_group='Copy Source')
+                      arg_group='Copy Source',
+                      help='A URL of up to 2 KB in length that specifies an Azure file or blob. '
```

Comment:
please adjust the help message. you need to add different help message for different command to be consistent with track1 interface. For example, `az storage blob copy start` vs `az storage blob incremental-copy start`

Reply:
also need to add backtick for https url for document issue.

Reply:
Good catch!  Actually, this example doesn't need to be added here. It already exists in the `_help` file, so I deleted it~